app/src/main/java/org/eyeseetea/malariacare/domain/entity/Form.java

```diff
@@ -27,66 +28,42 @@ public long getId() {
         return id;
     }
 
-    public void setId(long id) {
+    private void setId(long id) {
         this.id = id;
     }
 
     public String getName() {
         return name;
     }
 
-    public void setName(String name) {
+    private void setName(String name) {
         this.name = name;
     }
 
     public int getIndex() {
         return index;
     }
 
-    public void setIndex(int index) {
+    private void setIndex(int index) {
         this.index = index;
     }
 
     public Type getType() {
         return type;
     }
 
-    public void setType(Type type) {
+    private void setType(Type type) {
         this.type = type;
     }
 
     public Country getCountry() {
         return country;
     }
 
-    public void setCountry(Country country) {
+    private void setCountry(Country country) {
         this.country = country;
     }
 
-    @Override
-    public boolean equals(Object o) {
```

Comment:
Why are you remove equals and hashcode methods? These methods can be useful in many comparison situations. If there is no reason, create them again, please.

Reply:
Done!